src/api/app/views/webui/groups/_group_members.html.haml

```diff
@@ -6,33 +6,37 @@
     This group does not have members.
 - else
   #group-users
-    .d-flex.flex-wrap
+    .d-flex.flex-wrap.gap-2
```

Comment:
In the name of consistency I'd align it to `gap-4` (which I personally prefer), or the other way around (which I don't mind either btw) I'd change to `gap-2` also the [repository](https://github.com/openSUSE/open-build-service/pull/13826) and the [token](https://github.com/openSUSE/open-build-service/pull/13829) usage of the card component

Reply:
I'll with `gap-4`